aws/tagsPinPointApp.go

```diff
@@ -0,0 +1,133 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/pinpoint"
+	"github.com/hashicorp/terraform/helper/schema"
+	"log"
+	"regexp"
+)
+
+// getTags is a helper to get the tags for a resource. It expects the
+// tags field to be named "tags"
+func getTagsPinPointApp(conn *pinpoint.Pinpoint, d *schema.ResourceData, sn string) error {
```

Comment:
The `sn` parameter to this function appears to be unused and should be removed.

```suggestion
func getTagsPinPointApp(conn *pinpoint.Pinpoint, d *schema.ResourceData) error {
```

Reply:
Nice catch, Removed and updated function call.